### ISSUE TITLE
Buggy Unity Editor Transforms

### ISSUE THREAD

**Issue by linkoid** [BunyaPineTree/LethalCompany_ModelReplacementSDK#16]
Trying to rotate certain transforms in the Unity Editor is difficult or impossible when the OffsetBuilder is enabled.
This is because OffsetBuilder is forcing bones to be in a certain position every update before the parent's transformations can propagate to the children.

**Comment by BunyaPineTree** [comment]
If I'm not mistaken, didn't you mention in the discord channel that fixedupdate doesn't run in the editor? Wouldn't this mean that you couldn't update bone rotation offsets when not in playmode?

**Comment by linkoid** [comment]
Yeah, that pull request doesn't actually fix this problem, so I'm closing it.
